src/commands/autocomplete/values.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import {flags} from 'cli-engine-command'
+import AutocompleteBase from '.'
+import Plugins from '../../plugins'
+import ACCache from '../../cache'
+import path from 'path'
+
+export default class AutocompleteValues extends AutocompleteBase {
+  static topic = 'autocomplete'
+  static command = 'values'
+  static description = 'generates autocomplete values'
+  static hidden = true
+  static flags = {
+    // don't require cmd or flag
+    // we want it to run silently
+    // or autocomplete will use any
+    // flag errors as options
+    cmd: flags.string({description: '', char: 'c'}),
+    flag: flags.string({description: '', char: 'f'})
+  }
+
+  async run () {
+    try {
+      this.errorIfWindows()
+
+      if (this.flags.cmd) {
+        const plugins = new Plugins(this.out)
+        await plugins.load()
+        let Command = await plugins.findCommand(this.flags.cmd)
+        if (!Command || !this.flags.flag) this.out.error(`Command ${this.flags.cmd} not found`)
```

Comment:
@RasPhilCo could you split out the `this.flags.flag` check since the error will not make much sense in that case